crates/daphne-server/src/storage_proxy_connection/kv/mod.rs

```diff
@@ -212,13 +212,9 @@ impl<'h> Kv<'h> {
         let key = Self::to_key::<P>(key);
         tracing::debug!(key, "GET");
         match self.cache.read().await.get::<P>(&key) {
-            cache::GetResult::NotFound { read_through } => {
-                if !read_through {
-                    return Ok(None);
-                }
-            }
-            cache::GetResult::Found(t) => return Ok(Some(mapper(t))),
-            cache::GetResult::MismatchedType => {
+            cache::CacheResult::Miss => {}
+            cache::CacheResult::Hit(t) => return Ok(t.map(mapper)),
```

Comment:
I still think a code comment is needed to parse this, but if you disagree, disregard.